pyproject.toml

```diff
@@ -309,6 +309,7 @@ lint.select = [
     "PLR04",
     "PLR1",
     "RUF",  # ruff
+    "D",    # pydocstyle
```

Comment:
I was hoping we could keep the order of the rules in sync with https://docs.astral.sh/ruff/rules/ 
(Trying to tidy it up in https://github.com/teemtee/tmt/pull/2842/)

Reply:
Sure, fixed in 3a977e19